### ISSUE TITLE
Support for unicode whitespace

### ISSUE THREAD

**Issue by AurielleP** [spencermountain/compromise#759]
`\xa0` should either be replace with a regular space or explicitly trimmed/removed also when tokenizing

```
'Miami, FL 33178' // debug shows `'FL '` for that term
```

<img width="401" alt="Screen Shot 2020-06-04 at 8 57 23 AM" src="https://user-images.githubusercontent.com/7937261/83779974-6f620000-a641-11ea-9864-26b083f29056.png">

**Comment by spencermountain** [comment]
hey - is this just for the debug output?

**Comment by AurielleP** [comment]
@spencermountain Nope - the screenshot was just to demonstrate how its getting tokenized - so the issue is the `@tagName` formatting and other tagging isnt working quite right because it doesnt match if the non breaking space is on the end

**Comment by spencermountain** [comment]
ok good find. added support for this character in `13.3.1`

**Comment by MarkHerhold** [comment]
Chiming in with my two cents here -

I probably would leave data cleaning up to the end user (you are going to get all sorts of stuff from scraping web pages, MS Word docs, PDFs, etc), but if you do want compromise to be in the business of data cleaning, I'd look at starting with the [printable-characters](https://github.com/xpl/printable-characters) module.

For example, adapted from the README:
```
const { zeroWidthCharacters } = require ('printable-characters');
const s = '\u001b[106m' + 'foo' + '\n' + 'bar' + '\u001b[49m';
s.replace(zeroWidthCharacters, ''); // === 'foobar'
```

**Comment by spencermountain** [comment]
yeah, you're right.

**Comment by AurielleP** [comment]
@MarkHerhold I agree on the principle of data cleaning etc - and I dont think that spaces or newlines etc should be replaced with nothing - but I do think that `\xa0` should be treated like other spaces - or at least somehow made visible.

This type of issue mistakenly was instead interpreted as the library not behaving as expected because the non breaking space is invisible in most cases.

As a user, I would in most cases, likely expect, if not necessarily a replacement in the case of `\xa0`, but as a compromise (lol), that it would be treated the same as a regular space - and probably would live where other spaces live in the context of tokenization and phrases/terms i.e. found in the `pre` and `post` properties of a term etc

**Comment by spencermountain** [comment]
i've removed zero-width characters from normalized output, including normalized json output
Will be added to the next release. You can see the cursed string [here](https://github.com/spencermountain/compromise/blob/78c53b26f742bd033597a435f76beeb89c7023e3/tests/unicode.test.js#L22).
cheers

**Comment by spencermountain** [comment]
released as `13.8.0` - pls let me know if you find any other dark-magic
thanks!
